src/app/ReadHandler.h

```diff
@@ -323,10 +326,10 @@ class ReadHandler : public Messaging::ExchangeDelegate
     void GetSubscriptionId(SubscriptionId & aSubscriptionId) const { aSubscriptionId = mSubscriptionId; }
     AttributePathExpandIterator * GetAttributePathExpandIterator() { return &mAttributePathExpandIterator; }
 
-    /**
-     * Notify the read handler that a set of attribute paths has been marked dirty.
-     */
-    void SetDirty(const AttributePathParams & aAttributeChanged);
+    /// @brief Notifies the read handler that a set of attribute paths has been marked dirty. This will scehdule an engine run if
```

Comment:
```suggestion
    /// @brief Notifies the read handler that a set of attribute paths has been marked dirty. This will schedule a reporting engine run if
```